### PR TITLE
Update Auction

### LINKED ISSUE #68: Info Issue
A lot of the info commands still have the old bot commands.

### PR DESCRIPTION
**Pull request by Topkinsme**
Once done, this should fix #68

### CODE REVIEW

**Comment by Shepherdwhen** [comment]
hehe 69

File: _info/Vault [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Vault
 
-In this game, sharing your resources with your team is extremely important. You need to work with your team to manage your expenses in order to utilise it fully. You can use `!deposit` and `!withdraw `to deposit and withdraw cash to/from your team vault respectively. This is useful for when you need to save your cash in case you're about to die. Use `!vault` to check how much your vault holds. (Oh, and btw, you can use `!bal` to check your own balance.)
+In this game, sharing your resources with your team is extremely important. 
+You need to work with your team to manage your expenses in order to utilise it fully. 
+You can use `!vault deposit` and `!vault withdraw `to deposit and withdraw cash to/from your team vault respectively. 
+This is useful for when you need to save your cash in case you're about to die. Use `!vault view` to check how much your vault holds. 
+(Oh, and btw, you can use `!bal` to check your own balance.)
```
Comment (12Metatron) [inline _info/Vault:7]: this line probably needs rephrasing for obvious reasons

File: _info/Market [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Market
 
-Not to be mistaken for the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. But since there items are really powerful, there items are behind a heavy paywall. Use `!market` to check it out. You can use `!dismarket` to check your market level. You need to use `!upmarket` to upgrade your market. Use `!tmbuy` to buy the desired items. Note that Solos have the entire market unlocked for free. Also note that these items will only be used if you explicitly inform the hosts that you are using these items.
+Not to be mistaken for the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. But since there items are really powerful, there items are behind a heavy paywall. 
+Use `!market view` to check it out. You can use `!market level` to check your market level. You need to use `!upmarket` to upgrade your market. Use `!market buy` to buy the desired items. 
+Note that Solos have the entire market unlocked for free. Also note that these items will only be used if you explicitly inform the hosts that you are using these items.
```
Comment (12Metatron) [inline _info/Market:5]: should mention that this is replaced with black market if dead

File: _info/Forge [hunk 3]
```diff
@@ -1,3 +1,5 @@
 Forge
 
-The main source of income this game is called the forge. Every time a phase changed, every member of your team will get 100*<Your forge level> amount of cash. The forge level can be accessed by using `!disforge` and you can use `!upforge` to upgrade it! Doing this is very useful in the long run. Note that solo roles get a level 7 forge from the start.
+The main source of income this game is called the forge. Every time a phase changed, every member of your team will get 100*<Your forge level> amount of cash.
```
Comment (12Metatron) [inline _info/Forge:3]: "The main source of income this game is called the forge" -> "You teams main source of income comes from the forge" or something along those lines

**Review by 12Metatron — APPROVED** [review]

File: _info/Forge (hunk 3 above)
Comment (Topkinsme) [reply _info/Forge]: Fixed I think

File: _info/Market (hunk 2 above)
Comment (Topkinsme) [reply _info/Market:6]: Should be fixed.

File: _info/Vault (hunk 1 above)
Comment (Topkinsme) [reply _info/Vault]: I think it's fixed now

File: _info/Black_market [hunk 4]
```diff
@@ -1,3 +1,5 @@
 Black_Market
 
-Not to be mistaken for the Market or the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. Note that the black market is only visible to the people in the respawning phase, and all the items purchased there can only be used when in the respawning phase. All the other conditions of the market applies to the black market as well.
+Not to be mistaken for the Market or the Stock market, this market holds a Varity of useful items for anyone to use, which can drastically change the situation. 
+Note that the black market is only visible to the people in the respawning phase, and all the items purchased there can only be used when in the respawning phase. 
+All the other conditions of the market applies to the black market as well. Use `!market view` to view the black market.
```
Comment (12Metatron) [inline _info/Black_market:5]: i think it would help to specify that the commands are the same as regular market

**Review by 12Metatron — APPROVED** [review]
overalll though, good change. you can merge after you fix that last thing I commented on.

File: _info/Black_market (hunk 4 above)
Comment (Topkinsme) [reply _info/Black_market:5]: alright will fix

**Comment by Topkinsme** [comment]
I'm gonna hold out on merging for a few hours, incase some other issue is found

**Comment by Topkinsme** [comment]
Merging

**Pull request merged**